backend/src/main/java/co/inventorsoft/scripty/controller/MockRequestController.java

```diff
@@ -0,0 +1,76 @@
+package co.inventorsoft.scripty.controller;
+
+import co.inventorsoft.scripty.model.dto.MockRequestDto;
+import co.inventorsoft.scripty.model.dto.StringResponse;
+import co.inventorsoft.scripty.service.MockRequestService;
+import com.fasterxml.jackson.core.JsonParseException;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.core.env.Environment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.InvalidMimeTypeException;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.nio.charset.Charset;
+import java.nio.charset.IllegalCharsetNameException;
+import java.util.regex.Pattern;
+
+/**
+ * @author A1lexen
+ */
+
+@Api(description = "Operations with mock requests")
+@RestController
+public class MockRequestController {
+    @Autowired
+    MockRequestService requestService;
+
+    @Autowired
+    Environment environment;
+
+    @ApiOperation(value = "Create mock response with specified parameters")
+    @PostMapping(value="/mock_request")
+    public ResponseEntity createMockRequst(@Valid @RequestBody MockRequestDto requestDto) {
+        String token = requestService.createNewRequest(requestDto);
+
+        return ResponseEntity.status(HttpStatus.CREATED)
+                .body(new StringResponse(environment.getProperty("prefix.link") + "mock_request/" + token));
```

Comment:
You don't need to inject `Environment` to get single property. Please, use `@Value` annotation to get what you want